phonebook-api/src/main/java/com/telran/phonebookapi/controller/PhoneController.java

```diff
@@ -73,7 +73,7 @@ public void removeById(Authentication auth, @PathVariable int id) {
         phoneService.removeById(id);
     }
 
-    @GetMapping("/{contactId}")
+    @GetMapping("all/{contactId}")
```

Comment:
strictly speaking, the proper address is "/api/contact/{contactId}/phone"  - place it to the contact controller and fetch all the phones by simply calling "contact.getPhones()".
Also would be great, if you fir all such endpoints. If there are questions - please google "REST API" url naming